cfme/infrastructure/virtual_machines.py

```diff
@@ -684,6 +685,28 @@ def active(self):
             else:
                 return has_child(view.tree, '{} (Active)'.format(title), root_item)
 
+        @property
+        def size(self):
+            """
+            Check the shapshot size in the UI. So far available only in RHV and CFME > "5.11"
+
+            :returns the size of the snapshot
+            """
+            from cfme.infrastructure.provider.rhevm import RHEVMProvider
+            if not self.parent_vm.provider.one_of(RHEVMProvider):
+                raise Exception("Provider is not RHV, this feature is not available")
+            if self.parent_vm.appliance.version < "5.11":
+                raise Exception("This feature is available only starting in CFME 5.11")
+            title = getattr(self, self.parent_vm.provider.SNAPSHOT_TITLE)
+            view = navigate_to(self.parent_vm, 'SnapshotsAll')
```

Comment:
TODO: remove the duplicated snapshot lookup in all these methods by providing a snapshot details navigation destination.